hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/event/Event.java

```diff
@@ -35,6 +39,12 @@
  * @author Jay Shaughnessy
  * @author Lucas Ponce
  */
+@JsonTypeInfo(
```

Comment:
Nice annotation.
Note: This comment is not related to this PR but we have a custom deserializer that could be substituted by this annotation. This is a change with low priority and it's not a problem, but at some point (a spring clean or similar kind of task, it's something we can do).